workspace/build_target.go

```diff
@@ -156,10 +156,10 @@ func (bt BuildTarget) Build(runtimeCtx *runtime.Runtime, flags BuildFlags, packa
 
 	// Do this after the containers are up
 	for _, envString := range bt.EnvironmentVariables(runtimeCtx.EnvironmentData()) {
-		parts := strings.Split(envString, "=")
+		parts := strings.SplitN(envString, "=", 2)
```

Comment:
Optional: Perhaps this should be extracted out to a function to share with the code in #87?

Reply:
Makes sense